klaytn/src/adapter.ts

```diff
@@ -0,0 +1,131 @@
+import { Requester, Validator, AdapterError } from '@chainlink/external-adapter'
+import { Config, ExecuteWithConfig, ExecuteFactory, AdapterResponse } from '@chainlink/types'
+import { makeConfig } from './config'
+import { LegacyTransaction } from './types'
+import fs from 'fs'
+
+const Caver = require('caver-js')
+const caver = new Caver(process.env.URL)
```

Comment:
I'd recommend moving line 8 to inside of `klaytn/src/config.ts`'s `makeConfig`. We have an internal util that may help: 
```
import { util } from '@chainlink/ea-bootstrap'

// makeConfig
const util.getRequiredEnv('URL, prefix)
```
 which will throw if the URL is not provided is not provided. It can then be utilized from the Config object parameter on line 90. This means they will be logged when the External Adapter starts up (the secret will be hidden) which can be useful information when debugging.

Reply:
I modified the code to use `util` for env vars. However, I still don't see the flags in the log.
```
yarn start
yarn run v1.22.5
$ tsc -b
✨  Done in 1.79s.
yarn run v1.22.5
$ yarn server:dist
$ node -e 'require("./dist/index.js").server()'
{"message":"Listening on port 5002!","level":"info","instanceId":"102171fe-9e45-4f74-89cd-860ddcf3df27","timestamp":"2021-03-02T02:19:43.148Z"}
{"message":"API will be rate limited without an API key.","level":"warn","instanceId":"102171fe-9e45-4f74-89cd-860ddcf3df27","timestamp":"2021-03-02T02:19:47.292Z"}
{"message":"API will be rate limited without an API key.","level":"warn","instanceId":"102171fe-9e45-4f74-89cd-860ddcf3df27","timestamp":"2021-03-02T02:19:47.328Z"}
...
```